lib/dbservice_web/controllers/form_schema_controller.ex

```diff
@@ -0,0 +1,142 @@
+defmodule DbserviceWeb.FormSchemaController do
+  use DbserviceWeb, :controller
+
+  import Ecto.Query
+  alias Dbservice.Repo
+  alias Dbservice.Users
+  alias Dbservice.Users.User
+
+  action_fallback DbserviceWeb.FallbackController
+
+  use PhoenixSwagger
+
+  alias DbserviceWeb.SwaggerSchema.User, as: SwaggerSchemaUser
+  alias DbserviceWeb.SwaggerSchema.Common, as: SwaggerSchemaCommon
+
+  def swagger_definitions do
+    Map.merge(
+      Map.merge(
+        SwaggerSchemaUser.user(),
+        SwaggerSchemaUser.users()
+      ),
+      SwaggerSchemaCommon.group_ids()
+    )
+  end
+
+  swagger_path :index do
+    get("/api/user?full_name=Rahul Sharma")
+    response(200, "OK", Schema.ref(:Users))
+  end
+
+  def index(conn, params) do
+    query =
+      from m in User,
+        order_by: [asc: m.id],
+        offset: ^params["offset"],
+        limit: ^params["limit"]
+
+    query =
+      Enum.reduce(params, query, fn {key, value}, acc ->
+        case String.to_existing_atom(key) do
+          :offset -> acc
+          :limit -> acc
+          atom -> from u in acc, where: field(u, ^atom) == ^value
+        end
+      end)
+
+    user = Repo.all(query)
+    render(conn, "index.json", user: user)
+  end
+
+  swagger_path :create do
+    post("/api/user")
```

Comment:
this api seems to be incorrect api here. I think it would be:
`post("/api/form-schema")`